src/test/java/SetTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("set_size_함수_테스트")
+    @Test
+    void set_size_1() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+
+    @DisplayName("set_contains_함수_정상_테스트_ValueSource_사용")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void set_contains_1(int input) {
+        assertTrue(numbers.contains(input));
+    }
```

Comment:
assertj 를 활용하시면 다음과 같이 검증도 가능합니다!! 참고만 해주세요!! 

```suggestion
    void set_contains_1(int input) {
        assertThat(numbers).contains(input);
    }
```